opencti-platform/opencti-graphql/src/modules/notification/notification-domain.ts

```diff
@@ -232,7 +233,7 @@ export const triggerDelete = async (context: AuthContext, user: AuthUser, trigge
     throw ForbiddenAccess();
   }
   // If user is only organization admin, check if he has access on all targets
-  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && isUserHasCapability(user, VIRTUAL_ORGANIZATION_ADMIN)) {
+  if (!isAdminWithoutSettingsSetAccessCapability(user)) {
```

Comment:
should be `if (isAdminWithoutSettingsSetAccessCapability(user))`, without the `!`